spring-integration-core/src/main/java/org/springframework/integration/support/leader/LockRegistryLeaderInitiator.java

```diff
@@ -168,6 +174,16 @@ public LockRegistryLeaderInitiator(LockRegistry locks, Candidate candidate) {
 		this.candidate = candidate;
 	}
 
+	/**
+	 * Sets the {@link ExecutorService}
```

Comment:
I told already that method JavaDocs should go in the imperative style: https://github.com/spring-projects/spring-integration/pull/2357#discussion_r167612477.

Also would be great to mention what is default.

Reply:
I was following other setter docs in the class

```
/**
 * Sets the {@link LeaderEventPublisher}.
 * @param leaderEventPublisher the event publisher
 */
```

I will add to description
`where is not provided then a default of single thread Executor will be used`